valkyrie/lib/valkyrie/specs/shared_specs/file_characterization_service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'a Valkyrie::FileCharacterizationService' do
```

Comment:
You need another shared spec to ensure that a `file_node` is returned after characterization, since you're relying on its return value.